spec_1/The_Guessing_Game.js

```diff
@@ -0,0 +1,76 @@
+var readlineSync = require('readline-sync')
+var prompt = readlineSync.question
+let playAgain = true
+let floor = 1
+let ceiling = 100
+
+console.log(`Welcome to The Guessing Game\n I'm your host Trumpye Best`)
+
+const playGame = (floor, ceiling) => {
+
+  const generateRandomNumber = (floor, ceiling) => {
+    return Math.floor(Math.random() * ceiling) + floor
+  }
+
+  let userGuess
+
+  const getUserGuess = (floor, ceiling) =>{
+    userGuess = prompt(`Guess a number between ${floor} and ${ceiling}\n>>`)
+  }
+
+  while (userGuess < floor || userGuess > ceiling || userGuess === undefined){
+    getUserGuess(floor, ceiling)
+    if(userGuess === 'exit'){
+      playAgain = false
+    }
+    if(userGuess<floor||userGuess>ceiling){
+      console.log(`Please stay within the bounds of ${floor} and ${ceiling}`)
+    }
+    if(typeof userGuess !== 'number'){
+      console.log(`Please give a number`)
+    }
+  }
+
+  console.log(`Randomly generating a number between ${floor} and ${ceiling}...`)
+
+  const generatedNumber = generateRandomNumber(floor, ceiling)
+
+  const greaterThan = (guess) => {
+    return guess > generatedNumber
+  }
+
+  const lesserThan = (guess) => {
+    return guess < generatedNumber
+  }
+
+  const equalTo = (guess) => {
```

Comment:
not a big thing but you don't need the parenthesis with single arguments or the curly braces

``` const equalTo = guess => { return guess === generatedNumber } ```